src/config/default.yml

```diff
@@ -79,24 +79,22 @@ try:
       - taskId: '{{#as_slugid}}decision-task{{/as_slugid}}'
         task:
           created: '{{now}}'
-          deadline: '{{#from_now }}5 minutes{{/from_now}}'
+          deadline: '{{#from_now }}30 minutes{{/from_now}}'
           metadata:
             source: '{{{source}}}'
             owner: '{{owner}}'
-            name: Error message...
-            description: |
-              Error creating decision task...
+            name: Error Creating Decision Task
+            description: Error creating decision task
 
-          workerType: gecko-decision
+          workerType: "gecko-{{level}}-decision"
           provisionerId: aws-provisioner-v1
           routes:
-             - "tc-treeherder-stage.{{project}}.{{revision_hash}}"
-             - "tc-treeherder.{{project}}.{{revision_hash}}"
+             - "tc-treeherder.v2.{{project}}.{{revision_hash}}.{{pushlog_id}}"
```

Comment:
Ah, I want {{revision}} here -- {{revision_hash}} is a sha1 of the concatenation of all revisions in a push.